spec/text-editor-presenter-spec.coffee

```diff
@@ -2300,6 +2300,15 @@ describe "TextEditorPresenter", ->
 
             expect(decorationState[decoration3.id]).toBeUndefined()
 
+          it "updates when another gutter with higher priority is hidden", ->
+            editor.addGutter({name: 'test-gutter-1', priority: -150, visible: false})
```

Comment:
What about extracting a variable here, to make clear we are adding an hidden gutter? e.g.

```coffee
hiddenGutter = {name: 'any-name', priority: -150, visible: false}
editor.addGutter(hiddenGutter)
```